tests/test_advisory.py

```diff
@@ -68,3 +68,7 @@ def test_get_security_alerts(self):
         ads = self.me.get_advisories()
         for ad in ads:
             self.assertFalse(ad.has_blocking_secruity_alert(), f"advisory {ad.errata_id} has blocking security alerts")
+
+    def test_kernel_tag(self):
+       Advisory(errata_id=144853, impetus='image').check_kernel_tag()
```

Comment:
suggest to add assertion code to verify return value of func `check_kernel_tag `